### ISSUE TITLE
Python 3.6 Lambda created in Amplify CLI, but this runtime is deprecated by the Runtime deprecation policy 

### ISSUE THREAD

**Issue by sasssssssa** [aws-amplify/amplify-cli#10242]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-cli/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### Amplify CLI Version

8.0.1

### What operating system are you using?

Mac

### Did you make any manual changes to the cloud resources managed by Amplify? Please describe the changes made.

No

### Amplify Categories

function, api

### Amplify Commands

add, push

### Describe the bug

We found the the Python 3.6 Lambda created by Amplify CLI.
But, the Python3.6 in Lambda is deprecated in Phase 1 at July 18, 2022.
So, please upgrade Lambda Runtime, and tell us how to upgrade the Lambda created in the past.

Runtime deprecation policy
https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html
~~~
Runtime end of support dates
Name	Identifier	Operating system	Deprecation Phase 1	Deprecation Phase 2
...
Python 3.6
python3.6
Amazon Linux
July 18, 2022
Aug 17, 2022
~~~

I think it's the code that creates this lambda
https://github.com/aws-amplify/amplify-cli/blob/dfcfa5237d538088331e8a7c7d51a9932c879429/packages/amplify-graphql-searchable-transformer/src/cdk/create-cfnParameters.ts

### Expected behavior

This lambda should use a non-deprecated runtime

### Reproduction steps

The Python 3.6 Lambda is created by schema.graphq in a API Category.
```
type Student @model @searchable {
  name: String
  dateOfBirth: AWSDate
  email: AWSEmail
  examsCompleted: Int
}
```

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by josefaidt** [comment]
Hey @sasssssssa :wave: thanks for raising this! I've marked this as a duplicate bug of https://github.com/aws-amplify/amplify-cli/issues/10242 🙂

**Comment by andy-HM** [comment]
Hello, as of today with current CLI version 9.1.0, this is still an issue.  I this supposed to be considered fixed in 9.1.0?
